petcare-sample/b2c/mobile-app/petcare-with-sdk/README.md

```diff
@@ -14,34 +14,32 @@
 1.1 Setup the latest version of WSO2 Identity Sever.
 
 1.2 [Create a mobile application](https://is.docs.wso2.com/en/next/guides/applications/register-mobile-app/) (Un tick the PKCE `Mandayory` checkbox)
+- Add `wso2.apiauth.sample.android://login-callback` as the redirect URL of the application
 
 #### 2. Import the configurations
-2.1 Create `config.xml` file in `res/values`
+2.1 Go to the `<APP_CONTENT_LOCATION>/com/wso2_sample/api_auth_sample/util/Config.kt` file and change the following values according the application you created above,
 
 ```
-<resources>
-    <string name="oauth_client_base_url"> <BASE_URL> </string>
-    <string name="oauth_client_client_id"> <CLIENT_ID> </string>
-    <string name="oauth_client_redirect_uri">https://example-app.com/redirect</string>
-    <string name="oauth_client_scope">openid internal_login</string>
-    <string name="oauth_client_response_type">code</string>
-    <string name="oauth_client_response_mode">direct</string>
-    <string name="oauth_client_google_web_client_id"> <WSO2_CLIENT_ID_OF_GOOGLE> </string>
-    <string name="oauth_client_google_web_client_secret"> <WSO2_CLIENT_SECRET_OF_GOOGLE> </string>
-    <string name="data_source_resource_server_url"> <PET_RESOURCE_SERVER_URL> </string>
-</resources>
+private const val BASE_URL: String = <BASE_URL of the tenant>
+private const val DISCOVERY_URL: String = null
+private const val CLIENT_ID: String = "<CLIENT_ID of th>
```

Comment:
Shall we recheck this line?

Reply:
Fixed with https://github.com/wso2/samples-is/pull/469/commits/e14d666d71d9b6e1f167f63fb366b2668b7cf01e